ygen/gogen.go

```diff
@@ -1281,9 +1281,9 @@ func writeGoHeader(yangFiles, includePaths []string, cfg GeneratorConfig, rootNa
 	return common.String(), oneoff.String(), nil
 }
 
-// isScalarField determines which fields should be converted to pointers when
+// IsScalarField determines which fields should be converted to pointers when
 // outputting structs; this is done to allow checks against nil.
-func isScalarField(field *NodeDetails) bool {
+func IsScalarField(field *NodeDetails) bool {
```

Comment:
Should this be in `util` if it's public?

Reply:
This is operating on a `ygen` type though. It would later go into the `gogen` package.

Reply:
SGTM.